code/Calendar.php

```diff
@@ -276,19 +276,31 @@ public static function getWeekdayName($value, $width = 'wide', $locale = '', $st
     {
         $result = '';
         if ((!empty($value)) || ($value === 0) || ($value === '0')) {
-            $weekday = null;
-            if (is_int($value)) {
-                $weekday = $value;
-            } elseif (is_float($value)) {
-                $weekday = intval($value);
-            } elseif (is_string($value)) {
-                if (is_numeric($value)) {
+            switch (gettype($value)) {
```

Comment:
I checked the php manual changelog and I found this:
http://svn.php.net/viewvc?view=revision&revision=322892
Three years ago that warning was removed in the English version. Some of the other languages didn't updated their message. I think I'm fine with gettext. In the remote case that it'll change I'll update Punic.